cuebot/src/main/java/com/imageworks/spcue/dao/postgres/NestedWhiteboardDaoJdbc.java

```diff
@@ -210,19 +264,23 @@ public NestedGroup mapRow(ResultSet rs, int rowNum) throws SQLException {
     }
 
     private NestedJobWhiteboardMapper updateConnections(NestedJobWhiteboardMapper mapper) {
-        for (Map.Entry<String, List<String>> entry : mapper.childrenMap.entrySet()) {
-            NestedGroup group = mapper.groups.get(entry.getKey());
-            NestedGroupSeq.Builder childrenBuilder = NestedGroupSeq.newBuilder();
-            for (String childId : entry.getValue()) {
-                NestedGroup child = mapper.groups.get(childId);
-                child = child.toBuilder().setParent(group).build();
-                childrenBuilder.addNestedGroups(child);
-                mapper.groups.put(childId, child);
-            }
-            group = group.toBuilder()
-                    .setGroups(childrenBuilder.build())
-                    .build();
-            mapper.groups.put(entry.getKey(), group);
+        ArrayList<ChildrenEntry> orderedChildren = new ArrayList<>(mapper.childrenMap.values());
+        orderedChildren.sort(ChildrenEntry::compareTo);
+
+        for (ChildrenEntry entry : orderedChildren) {
+           logger.debug(entry.toString());
```

Comment:
There are a couple of debug messages in this change -- are they useful or should we remove those? Here for example we probably need to add some more context to the log message to make it useful.

Reply:
Yes those debug messages can be removed - will do that.